test/cmd/basicresources.sh

```diff
@@ -338,7 +338,6 @@ os::cmd::expect_failure_and_text "oc new-project --token="$( oc sa get-token bui
 
 # test oc projects
 os::cmd::expect_failure_and_text 'oc projects test_arg' 'no arguments'
-os::cmd::expect_success_and_text 'oc projects' 'You have access'
```

Comment:
Then create two projects here and do the same thing.  Knowing that we are properly handling multiple projects has value.